servers/SCsub

```diff
@@ -20,8 +20,7 @@ if not env["disable_navigation_2d"]:
 if not env["disable_physics_2d"]:
     SConscript("physics_2d/SCsub")
 
-if not env["disable_navigation_3d"]:
-    SConscript("navigation_3d/SCsub")
+SConscript("navigation_3d/SCsub")
```

Comment:
Might need to be done for the 2D server as well, would need to be tested with `disable_navigation_2d` specifically